internal/driver/daemon.go

```diff
@@ -384,11 +384,19 @@ func (r *RegistryDefault) streamInterceptors(ctx context.Context) []grpc.StreamS
 	return is
 }
 
-func (r *RegistryDefault) ReadGRPCServer(ctx context.Context) *grpc.Server {
-	s := grpc.NewServer(
+func (r *RegistryDefault) newGrpcServer(ctx context.Context) *grpc.Server {
+	opts := []grpc.ServerOption{
 		grpc.ChainStreamInterceptor(r.streamInterceptors(ctx)...),
 		grpc.ChainUnaryInterceptor(r.unaryInterceptors(ctx)...),
-	)
+	}
+	if r.grpcTransportCredentials != nil {
+		opts = append(opts, grpc.Creds(r.grpcTransportCredentials))
+	}
```

Comment:
We should rather enable TLS on the multiplexer (if that is even possible?)

Reply:
Yes, it is possible if we mux on the established HTTP connection, as seen here:

https://github.com/google/keytransparency/pull/205/commits/ad24e6d02c6b2cd5318183667cf995b56da0a42d#diff-fca3329b29f1d82d36a43d5e6f37764d88bf80c4ad17f769c7a3c135860d0981R121-R125

Reply:
WDYT, should we push this into a follow-up issue? TLS support in Keto would be a new feature.

Reply:
Yes separate PR sounds good :+1: 
Should we merge this one after TLS or before?

Reply:
I'd merge now and do TLS as a separate feature.